.infrastructure/helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{.Values.todoapp.RBAC.name}}
+  namespace: {{.Values.todoapp.namespace}}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{.Values.todoapp.namespace}}
+  name: {{.Values.todoapp.RBAC.name}}
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{.Values.todoapp.RBAC.name}}
+  namespace: {{.Values.todoapp.namespace}}
```

Comment:
Ensure that `{{.Values.todoapp.namespace}}` is defined in your `values.yaml` file for the `todoapp`. If it's not defined, it could lead to deployment errors.